<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INVIND760" failures="1" tests="16" name="testCases.TC_005_alt_in_image_test" time="392.448" errors="0" timestamp="27 Apr 2023 08:52:50 GMT" skipped="0">
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="21.415"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="45.219">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at testCases.TC_005_alt_in_image_test.TC_005_alt_in_image(TC_005_alt_in_image_test.java:116)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_005_alt_in_image -->
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="20.882"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="28.441"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="29.087"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="19.575"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="34.193"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="20.216"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="20.054"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="19.042"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="29.552"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="19.881"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="28.390"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="19.312"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="24.885"/>
  <testcase classname="testCases.TC_005_alt_in_image_test" name="TC_005_alt_in_image" time="12.304"/>
</testsuite> <!-- testCases.TC_005_alt_in_image_test -->
